cmd/ct_monitor/main.go

```diff
@@ -59,6 +60,11 @@ func main() {
 		logInfoFileName := fmt.Sprintf("%s.txt", logInfoFileName)
 		flags.LogInfoFile = logInfoFileName
 	}
+	if !flags.Once {
+		if err := server.StartMetricsServer(flags.MonitorPort); err != nil {
+			log.Fatalf("Failed to start Prometheus metrics server: %v", err)
+		}
+	}
```

Comment:
this seems to be a common operation to all monitors, we might just move it in the `MonitorLoop` function